From: Jiri Slaby <jslaby@suse.cz>
Subject: mark printing_task kthread with kgr_task_safe
Patch-mainline: Not yet, depends on printk and kgr patches
References: bnc#744692, bnc#789311

If we didn't do this, we would never finish the kgraft patching.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 kernel/printk/printk.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -2763,6 +2763,7 @@ static int printk_kthread_func(void *dat
 			schedule();
 
 		__set_current_state(TASK_RUNNING);
+		klp_kgraft_mark_task_safe(current);
 		/*
 		 * Avoid an infinite loop when console_unlock() cannot
 		 * access consoles, e.g. because console_suspended is
